addons/mayastor/disable

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -e
+
+source $SNAP/actions/common/utils.sh
+KUBECTL="$SNAP/kubectl --kubeconfig=${SNAP_DATA}/credentials/client.config"
+
+echo "Disabling Mayastor"
+
+# Delete the resources
+$KUBECTL delete -f https://raw.githubusercontent.com/openebs/mayastor-control-plane/master/deploy/operator-rbac.yaml --force || true
+$KUBECTL delete -f https://raw.githubusercontent.com/openebs/mayastor-control-plane/master/deploy/mayastorpoolcrd.yaml --force || true
+$KUBECTL delete -f https://raw.githubusercontent.com/openebs/mayastor/master/deploy/nats-deployment.yaml --force || true
+$KUBECTL delete -f https://raw.githubusercontent.com/openebs/mayastor/develop/deploy/etcd/storage/localpv.yaml --force || true
+
+# Using our sts without affinity
+$KUBECTL delete -f "${CURRENT_DIR}/etcd.yaml" --force || true
+
+$KUBECTL delete -f https://raw.githubusercontent.com/openebs/mayastor/master/deploy/etcd/svc.yaml --force || true
+$KUBECTL delete -f https://raw.githubusercontent.com/openebs/mayastor/master/deploy/etcd/svc-headless.yaml --force || true
+$KUBECTL delete -f https://raw.githubusercontent.com/openebs/mayastor/master/deploy/csi-daemonset.yaml --force || true
+$KUBECTL delete -f https://raw.githubusercontent.com/openebs/mayastor-control-plane/master/deploy/core-agents-deployment.yaml --force || true
+$KUBECTL delete -f https://raw.githubusercontent.com/openebs/mayastor-control-plane/master/deploy/rest-deployment.yaml --force || true
+$KUBECTL delete -f https://raw.githubusercontent.com/openebs/mayastor-control-plane/master/deploy/rest-service.yaml --force || true
+$KUBECTL delete -f https://raw.githubusercontent.com/openebs/mayastor-control-plane/master/deploy/csi-deployment.yaml --force || true
+$KUBECTL delete -f https://raw.githubusercontent.com/openebs/mayastor-control-plane/master/deploy/msp-deployment.yaml --force || true
+$KUBECTL delete -f https://raw.githubusercontent.com/openebs/mayastor/master/deploy/mayastor-daemonset.yaml --force || true
+
+# Delete the underlying storage file
+
+run_as_sudo losetup -d /dev/loop666
```

Comment:
Why is this hardcoded to `loop666`, cant we have it such that we use the next available device as given by losetup